src/ui/server/helpers/summary-lists/generate-field-group-item/index.test.ts

```diff
@@ -30,6 +30,24 @@ describe('server/helpers/summary-lists/generate-field-group-item', () => {
       expect(result).toEqual(expected);
     });
 
+    describe('when the number 0 is passed', () => {
+      it('should return an object with correct fields', () => {
+        const mockSubmittedDataNumber = { fieldA: 0 };
+
+        mockInput.data = mockSubmittedDataNumber;
```

Comment:
Can we rename to `mockSubmittedDataNumber` to `mockSubmittedData`, making it a `let` at the top?